KsApi/GraphAPI.swift

```diff
@@ -578,182 +578,6 @@ public enum GraphAPI {
     }
```

Comment:
This file gets re-gened every time `KsAPI` is rebuilt, so can't avoid it being a change.

Reply:
we might consider putting this in our .gitignore to avoid confusion. We intentionally update this file as needed, when we support a new mutation for example. is there a reason we need to update this file on every build?